test/both.js

```diff
@@ -1,6 +1,7 @@
 var R = require('..');
 var eq = require('./shared/eq');
 
+var S = require('sanctuary');
```

Comment:
This should probably come first, since it's an external dependency. A few months away from Plaid and look what has become of you. :stuck_out_tongue_winking_eye: